lib/polly.ml

```diff
@@ -40,14 +40,19 @@ end
 
 type t = Unix.file_descr (* epoll fd *)
 
-external caml_polly_add : t -> Unix.file_descr -> Events.t -> unit
+external caml_raise_unix_error : string -> 'a = "caml_raise_unix_error"
```

Comment:
I don't see why we need this. We can raise an exception simply from OCaml without calling into C first.

Reply:
Because C will get the errno last set by the unix call and position it in the exception. 
We probably should use that for eventfd though.

Reply:
By the way, why to you define __FUNCTION__ in eventfd.read and write ?

Reply:
`__FUNCTION__` is not defined in OCaml 4.08, unfortunately. I could have used `__LOC__` instead.